### PR TITLE
[JBPM-7281] Stunner - New Marshallers fail migration tests (throwing, boundary, basic)

### PR DESCRIPTION
**Pull request by evacchi**
collective PR for all these failing tests

Only one test is still failing, and that's because AssignmentsInfo still compares its string representation. All the required fields are present in the marshalled/unmarshalled version

@romartin @hasys please take a look

### CODE REVIEW

**Comment by evacchi** [comment]
Jenkins execute full downstream build

**Comment by evacchi** [comment]
@jomarko thanks, done

**Comment by hasys** [comment]
Hi @evacchi, @jomarko yes I will take a look, but sorry for now GA has higher priority so sorry for the delays :(

**Comment by romartin** [comment]
Jenkins please retest this

**Comment by romartin** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/service/diagram/marshalling/tasks/Task.java [hunk 1]
```diff
@@ -56,7 +55,7 @@
     public static List<Object[]> marshallers() {
         return Arrays.asList(new Object[][]{
                 // New (un)marshaller is disabled for now due to found incompleteness
```
[the remark this answers is not in the document]
Comment (evacchi) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/service/diagram/marshalling/tasks/Task.java]: ok

**Comment by evacchi** [comment]
addressed @hasys comment.

**Comment by evacchi** [comment]
Jenkins execute full downstream build

**Pull request merged**